framework/src/modules/chain/chain.js

```diff
@@ -45,6 +44,7 @@ if (typeof gc !== 'undefined') {
 	}, 60000);
 }
 
+let blockReward;
```

Comment:
prefer const in the scope

Reply:
This variable needs to be assigned when initializing the module.

Reply:
Moved it to a class variable